openedx_webhooks/decorators.py

```diff
@@ -0,0 +1,54 @@
+"""
```

Comment:
Searching PyPI under the "right" name ( :) ) finds a number of options: https://pypi.python.org/pypi?%3Aaction=search&term=cache%20decorator&submit=search